src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <div class="wrapper">
+      <form
+        action="#"
+        data-qa="big"
+        class="search-bar search-bar--big">
```

Comment:
The form tag is self-closed, which is not valid in HTML5. Please use a separate closing tag `</form>` to ensure compatibility across browsers.